velox/docs/functions/presto/conversion.rst

```diff
@@ -448,8 +450,22 @@ consistent with other supported cases of cast.
 
 ::
 
+  SELECT cast('infinity' as real); -- Infinity
+  SELECT cast('-infinity' as real); -- -Infinity
+  SELECT cast('inf' as real); -- Infinity
   SELECT cast('InfiNiTy' as real); -- Infinity
+  SELECT cast('INFINITY' as real); -- Infinity
   SELECT cast('nAn' as real); -- NaN
+  SELECT cast('nan' as real); -- NaN
```

Comment:
presto:test> SELECT cast('nan' as real);
Query 20231120_033942_00035_a9j2g failed: Cannot cast 'nan' to REAL